Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -617,37 +621,30 @@ def splunk_submit_event_hec_command():
         demisto.results('The event was sent successfully to Splunk.')
 
 
-def splunk_edit_notable_event_command(proxy):
-    if not proxy:
-        os.environ["HTTPS_PROXY"] = ""
-        os.environ["HTTP_PROXY"] = ""
-        os.environ["https_proxy"] = ""
-        os.environ["http_proxy"] = ""
-    baseurl = 'https://' + demisto.params()['host'] + ':' + demisto.params()['port'] + '/'
-    username = demisto.params()['authentication']['identifier']
-    password = demisto.params()['authentication']['password']
-    auth_req = requests.post(baseurl + 'services/auth/login',
-                             data={'username': username, 'password': password, 'output_mode': 'json'},
-                             verify=VERIFY_CERTIFICATE)
+def splunk_edit_notable_event_command(proxy, service, auth_token):
+    params = demisto.params()
+    base_url = 'https://' + params['host'] + ':' + params['port'] + '/'
+    sessionKey = service.token if not auth_token else None
 
-    sessionKey = auth_req.json()['sessionKey']
```

Comment:
You are saying this is redundent?

Reply:
yes, this function was copied from https://gist.github.com/LukeMurphey/b45a5425572deffd2e955e460f261dbf but its a wrong use, we already authenticate in the `main` function, so no need to re-authenticate. The Splunk `Service` classes does this authentication in the `login` method:
![image](https://user-images.githubusercontent.com/53565845/111479220-702fd280-8739-11eb-9c58-c1faa206b9ca.png)